src/ff7/widescreen.cpp

```diff
@@ -322,6 +322,12 @@ void Widescreen::init()
 {
     loadConfig();
     loadMovieConfig();
+    if (widescreen_ar == WIDE_16X10)
+    {
+        wide_viewport_x = wide_16x10_viewport_x;
+        wide_viewport_width = wide_16x10_viewport_width;
+        wide_game_width = wide_16x10_game_width;
```

Comment:
Can you please set the values directly instead of creating variables that are used only once in the entire code lifecycle?